js/loader.js

```diff
@@ -54,6 +54,16 @@ var Loader = (function () {
 
 		// Notify core of loaded modules.
 		MM.modulesStarted(moduleObjects);
+
+		// Starting modules also hides any modules that have requested to be initially hidden
+		for (let n in moduleObjects) {
+			let thisModule = moduleObjects[n];
```

Comment:
i think you could write it like this:
for (let thisModule of moduleObjects) {

Reply:
I.....yes.
Dammit.

:)